src/controllers/category.js

```
@@ -0,0 +1,23 @@
+
+const createCategory = async (req, res) => {
+  const categoryData = req.body
+
+  const newCategory = await categoryService.createCategory(categoryData)
+
+  res.status(201).json(newCategory)
+}
+
+const getCategories = async (req, res) => {
+  const { limit = DEFAULT_CATEGORY_LIMIT, skip = DEFAULT_CATEGORY_SKIP, sort, categories, name } = req.query
+
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)